### PR TITLE
Update readme #304

### PR DESCRIPTION
**Pull request by ocramz**
* add badges, contributing guidelines, contributor image wall
* fix code examples
* update cabal file and changelog to reflect recent changes in CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Data.Monoid (mconcat)
 
 main = scotty 3000 $
     get "/:word" $ do
-        beam <- param "word"
+        beam <- captureParam "word"
```
Comment (lokeshmohanty) [inline README.md:13]: Any reason to replace `param` with `captureParam`?

**Comment by ocramz** [comment]
yep! 'param' is deprecated since 0.20

**Comment by lokeshmohanty** [comment]
Thanks!
I was confused due to an error saying that `captureParam' doesn't exist and also it didn't show up on `hoogle'.

**Comment by ocramz** [comment]
ah, I now see hoogle still points to 0.12 , which is unfortunate. I suspect
this is because scotty-0.20 hasn't made it to a stackage LTS release yet.

in the meantime, consider if you're able to upgrade to > 0.20 ! a number of
papercuts have been fixed, with minimal breakage.

**Comment by lokeshmohanty** [comment]
Thanks a lot! Will do.

**Pull request merged**